Dialog/RoomConfigDialog.ui

```diff
@@ -155,103 +213,110 @@
-          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Use existing Layer 0&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Select the foreground layer pointer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
-         </property>
+        <widget class="QComboBox" name="ComboBox_LayerPriority"/>
+       </item>
+       <item>
+        <widget class="QLabel" name="roomWidthLabel">
          <property name="text">
-          <string>Use existing Layer 0:</string>
+          <string>Room Width:</string>
          </property>
         </widget>
        </item>
        <item>
-        <widget class="QComboBox" name="ComboBox_Layer0Picker">
-         <property name="enabled">
-          <bool>false</bool>
+        <widget class="QSpinBox" name="SpinBox_RoomWidth">
+         <property name="minimum">
+          <number>19</number>
+         </property>
+         <property name="maximum">
+          <number>255</number>
          </property>
         </widget>
        </item>
        <item>
-        <widget class="QCheckBox" name="CheckBox_Layer0AutoScroll">
-         <property name="enabled">
-          <bool>false</bool>
-         </property>
+        <widget class="QLabel" name="roomHeightLabel">
          <property name="text">
-          <string>Enable Layer 0 Autoscroll</string>
+          <string>Room Height:</string>
          </property>
         </widget>
        </item>
        <item>
-        <widget class="QCheckBox" name="CheckBox_Layer2Enable">
-         <property name="toolTip">
-          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Enable Layer 2&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Enable layer X to set its mapping type from 0x00 to 0x01.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Layer X can be Layer 0 or Layer 2.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+        <widget class="QSpinBox" name="SpinBox_RoomHeight">
+         <property name="minimum">
+          <number>14</number>
          </property>
-         <property name="text">
-          <string>Enable Layer 2</string>
+         <property name="maximum">
+          <number>255</number>
          </property>
         </widget>
        </item>
        <item>
-        <widget class="QLabel" name="layerPriorityLabel">
-         <property name="toolTip">
-          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Layer Priority&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;There's several available priority settings that Wario Land 4's game engine supports, you can use one of them.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
-         </property>
+        <widget class="QLabel" name="label_LayerGfxEffect01">
          <property name="text">
-          <string>Layer Priority:</string>
+          <string>Layer graphic effect flag 1(Hex):</string>
```

Comment:
For these descriptions let's add a space before "(Hex)"

Reply:
Done.